content/blog/2020/07/2020-07-24-custom-distribution-service-phase-2.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 2 Blogpost"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+- packaging
+- platform-sig
+author: sladyn98
+opengraph:
+  image: /images/gsoc/cds_gsoc.png
+---
+
+
+Hello everyone,
+It is time to wrap up another successfull phase for the link:https://github.com/jenkinsci/custom-distribution-service[cutsom distribution service project], we have incorporated most of the features that we had planned at the start of the phase. It has been an immense learning curve for me and the entire team.
```

Comment:
Please separate each sentence on its own line. Long lines are hard to review.

Reply:
Got it :+1: